### DISCUSSION TITLE
Retrieve results from a flow that ran in the past

### DISCUSSION THREAD

**Discussion by multimeric** [PrefectHQ/prefect#5383] [Q&A]
I have a workflow that I want to unit test. For this, I'm hoping to run each task in isolation, but to do that I need the output from the previous step, which is present because I have enabled checkpointing. This isn't trivial to infer from the filesystem, thus it would be nice to be able to load previous workflow metadata in order to find output files from a previous run. How can I do this?

**Comment by zanieb** [comment]
Hi! Check out https://docs.prefect.io/api/latest/tasks/prefect.html#get-task-run-result and https://docs.prefect.io/orchestration/flow-runs/task-runs.html#inspecting-task-runs
